Misc/Copy-ItemFromBcContainer.ps1

```diff
@@ -0,0 +1,66 @@
+﻿<# 
+ .Synopsis
+  Copy an item (file or folder) from a container to the host system.
+ .Description
+  Copy an item (file or folder) from a container to the host system.
+ .Parameter containerName
+  Name of the container from which you want to copy the item.
+ .Parameter containerPath
+  Path to the item in the container which has to be copied.
+ .Parameter localPath
+  Path on the host where the item will be placed.
+  If the source item is a folder, 'localPath' will be created (or treated) as a folder and all items inside the source folder will be copied to it.
+ .Example
+  Copy-ItemFromBcContainer -containerName test2 -containerPath "c:\run\my" -localPath "c:\temp\copy-of-my"
+#>
+function Copy-ItemFromBcContainer {
+    Param (
+        [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
+        [Parameter(Mandatory = $true)]
+        [string] $containerPath,
+        [Parameter(Mandatory = $false)]
+        [string] $localPath = $containerPath
+    )
+
+    $telemetryScope = InitTelemetryScope -name $MyInvocation.InvocationName -parameterValues $PSBoundParameters -includeParameters @()
+    try {
+
+        if (!(Test-BcContainer -containerName $containerName)) {
+            throw "Container $containerName does not exist"
+        }
+        Write-Host "Copy '$containerPath' from container $containerName to '$localPath' on host"
```

Comment:
I think we should test whether localpath exists here and throw early

Reply:
The idea is that local path actually _can_ exist. If it does, the contents from source are added _inside_ target. I am still not sure whether that's a good idea or failing is better.

Reply:
Ahh, I didn't notice that CreateDirectory only creates the directory if it doesn't already exist.